www/src/pages/components/text-input/android/index.mdx

```diff
@@ -0,0 +1,50 @@
+---
+title: Text Input
+description: Form inputs with sizes and style variations.
+---
+
+import { graphql } from 'gatsby';
+import { ComponentHeader } from 'components/thumbprint-components';
+
+<ComponentHeader data={props.data} />
+
+### Components
+
+There are four types of Text Input components:
+
+-   `ThumbprintTextInput` : A single line `ThumbprintEditText` with no drawables. While you can use `ThumbprintTextInputWithDrawables` without defining any drawables in place of this View, it is recommended to use this View if you do not plan on defining drawables. This is because `ThumbprintTextInputWithDrawables` has tap event listeners which will do unncessary work if no drawables are defined.
+-   `ThumbprintTextInputWithDrawables` : A single line `ThumbprintEditText` with drawables. Contains logic for capturing clicks on drawables (both start and end). Can expose drawable tap events through `.uiEvents()`.
+-   `ThumbprintTextArea` : A multi-line ThumbprintEditText. For iOS and web parity, this is referred to as a TextArea.
```

Comment:
Should this go under [documentation for `TextArea`](https://thumbprint.design/components/text-area/react/) component instead? 🤔

Reply:
Done.